cranelift/codegen/meta/src/cdsl/instructions.rs

```diff
@@ -301,7 +302,9 @@ fn verify_format(inst_name: &str, operands_in: &[Operand], format: &InstructionF
         if operand.is_value() {
             num_values += 1;
         }
-        if operand.is_immediate_or_entityref() {
+        if operand.kind.is_block() {
+            num_blocks += 1;
+        } else if operand.is_immediate_or_entityref() {
```

Comment:
The changes in this file aren't necessary for this PR, but did show up as necessary when making changes to the old form of `br_table`.